docs/tox.ini

```
@@ -56,8 +56,15 @@ deps =
   sling
 
   # Community integrations
+  git+https://github.com/dagster-io/community-integrations.git\#egg=dagster-iceberg&subdirectory=libraries/dagster-iceberg
   git+https://github.com/dagster-io/community-integrations.git\#egg=dagster-polars&subdirectory=libraries/dagster-polars
 
+  # Optional dependencies for dagster-iceberg
+  getdaft>=0.3.0
+  pandas>=2.0.0
+  polars>=1.0.0
+  pyspark[connect]>=3.4.0
```

```suggestion
  git+https://github.com/dagster-io/community-integrations.git\#egg=dagster-iceberg[daft,pandas,polars,spark]&subdirectory=libraries/dagster-iceberg
  git+https://github.com/dagster-io/community-integrations.git\#egg=dagster-polars&subdirectory=libraries/dagster-polars
```
didn't work (e.g. `daft` wasn't getting installed), hence this workaround. I did some digging to see if can support PEP-508 dependencies with `tox`, but it's not 100% clear how to do it in the context of the current setup.